packages/pyright-internal/src/localization/package.mb.en.json

```diff
@@ -0,0 +1,15 @@
+{
+    "Diagnostic": {
+        "booleanIsLowerCase": "\"{name}\" is not defined, did you mean \"{booleanName}\"?",
```

Comment:
Let's add the common errors that we didn't change the text of, as we want them to all be in a single file for translation purposes.

Reply:
This seems fairly arbitrary on my part, but these are the errors I commonly see - [`edd1ad4`](https://github.com/microbit-foundation/pyright/pull/10/commits/edd1ad4c1bf21e70c8e633a352832e51836162da).

Reply:
Let's discuss the text with Giles. Please can you add some more context for the "Expected expression" scenarios? Perhaps it is just very general but would be good to confirm.